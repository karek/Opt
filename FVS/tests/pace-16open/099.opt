8
18 22 21 0 25 20 24 8 
