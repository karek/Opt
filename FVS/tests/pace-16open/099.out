8
22 18 21 20 24 15 26 25 
